crates/e2e/tests/e2e/services.rs

```diff
@@ -302,7 +308,10 @@ impl OrderbookServices {
         LimitOrderQuoter {
             limit_order_age: chrono::Duration::seconds(15),
             loop_delay: Duration::from_secs(1),
-            quoter,
+            quoter: Arc::new(FixedFeeQuoter {
+                quoter,
+                fee: 1_000.into(),
+            }),
```

Comment:
I would actually like to see this quote fees using the baseline quoter.

Its still not 100% clear to me exactly where the rounding issue is coming from.

One concern that I have is that it actually comes from the clearing price calculation related to the limit prices and not the Naive solver.

Reply:
> Its still not 100% clear to me exactly where the rounding issue is coming from.

What part of the explanation do you not find convincing?
Note that when the test still set a `surplus_fee` of 0 the settlement contract was short of exactly 1 wei of the token swapped on uniswap. That's the exact behavior explained in the linked [PR](https://github.com/gnosis/gp-v2-services/pull/938).
I went a bit more into the details for the 0 `surplus_fee` case [here](https://github.com/cowprotocol/services/pull/775#issuecomment-1315120726) in case you missed that.

> And, if we use this fixed fee, it might be harder to catch regressions over there.

Not sure how a variable fee would help with that case.